packages/framework/attributor/src/mixinAttributor.ts

```diff
@@ -147,17 +145,38 @@ export const mixinAttributor = (Base: typeof ContainerRuntime = ContainerRuntime
 			)) as ContainerRuntimeWithAttributor;
 			runtime.runtimeAttributor = runtimeAttributor as RuntimeAttributor;
 
+			const logger = ChildLogger.create(runtime.logger, "Attributor");
+			const mc = loggerToMonitoringContext(context.taggedLogger);
+
+			const shouldTrackAttribution = mc.config.getBoolean(enableOnNewFileKey) ?? false;
+			if (shouldTrackAttribution) {
+				(context.options.attribution ??= {}).track = true;
+				// Add telemetry to track if the storage of attribution data was enabled in configuration or not
+				logger.sendTelemetryEvent({ eventName: "load", attributionEnabledInConfig: true });
+			}
+
 			// Note: this fetches attribution blobs relatively eagerly in the load flow; we may want to optimize
 			// this to avoid blocking on such information until application actually requests some op-based attribution
 			// info or we need to summarize. All that really needs to happen immediately is to start recording
 			// op seq# -> attributionInfo for new ops.
-			await runtime.runtimeAttributor.initialize(
-				deltaManager,
-				audience,
-				baseSnapshot,
-				async (id) => runtime.storage.readBlob(id),
-				shouldTrackAttribution,
+			await PerformanceEvent.timedExecAsync(
+				logger,
+				{
+					eventName: "initialize",
+					attributionEnabledInDoc: runtime.runtimeAttributor.isEnabled,
```

Comment:
`isEnabled` doesn't accurately reflect whether attribution is enabled until after initialize is called. You should use the perfEnd event and explicitly end it with this extra property post-initialization.